validator/sawtooth_validator/execution/scheduler_parallel.py

```diff
@@ -649,6 +649,12 @@ def set_transaction_execution_result(
             if txn_signature not in self._scheduled:
                 raise SchedulerError(
                     "transaction not scheduled: {}".format(txn_signature))
+
+            if txn_signature not in self._batches_by_txn_id:
+                LOGGER.debug('Received result for %s, but was unscheduled',
```

Comment:
This feels like it will clutter the logs.

Reply:
I was using the same logging strategy from the serial scheduler.  I could remove it from here and the serial scheduler.

Reply:
They could also be replaced with metrics.